repose-aggregator/components/filters/cors-filter/src/main/scala/org/openrepose/filters/cors/CorsFilter.scala

```diff
@@ -61,6 +61,15 @@ class CorsFilter @Inject()(configurationService: ConfigurationService)
   }
 
   override def doFilter(servletRequest: ServletRequest, servletResponse: ServletResponse, filterChain: FilterChain): Unit = {
+    if (!isInitialized) {
+      logger.error("Filter has not yet initialized... Please check your configuration files and your artifacts directory.")
+      servletResponse.asInstanceOf[HttpServletResponse].sendError(500)
+    } else {
+      doFilterGuarded(servletRequest, servletResponse, filterChain)
```

Comment:
This could be a single method, but it would make the diff huge due to everything being indented one level into the `else` clause. The easy compromise was to simply guard the original method. It is just as easy to remove the guard and indent, it just makes the review more difficult.